tests/test_operator.py

```diff
@@ -430,19 +433,22 @@ def test_directly_indexed_expression(self, fa, ti0, t0, exprs):
         assert trees[1][-1].nodes[0].expr.rhs == eqs[1].rhs
 
 
+@pytest.mark.skipif(configuration['backend'] != 'foreign',
```

Comment:
doesn't this line do its purpose: https://github.com/opesci/devito/blob/80b4d6dbae041ec2d78be99323d4d7f806af1ae0/.travis.yml#L54